Find a functionally equivalent code:android.view.PointerIcon.load:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Loads the bitmap and hotspot information for a pointer icon, if it is not already loaded.
 * Returns a pointer icon (not necessarily the same instance) with the information filled in.
 *
 * @param context The context.
 * @return The loaded pointer icon.
 *
 * @throws IllegalArgumentException if context is null.
 * @see #isLoaded()
 * @hide
 */

@@ -7,12 +7,12 @@
     }
     PointerIcon result = new PointerIcon(mStyle);
     result.mSystemIconResourceId = mSystemIconResourceId;
-    result.loadResource(context.getResources(), mSystemIconResourceId);
+    result.loadResource(context, context.getResources(), mSystemIconResourceId);
     return result;
 }

